app/frontend/styles/components/_paginated_filter.scss

```diff
@@ -80,6 +80,17 @@
   }
 }
 
+// Fix for fixed position of layout filters
+.js-enabled .moj-filter-layout__filter {
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+  }
```

Comment:
Do we need to do this for tablet? I haven't managed to see an issue on tablet sizes.

Reply:
On my local there were some problems on iPad mini and other small tablets formats.